mirth.rb

```diff
@@ -28,8 +28,8 @@ def add_data
 router = ActionDispatch::Routing::RouteSet.new
 
 router.draw do
-  get '/show/data', to: DailyDataController.action(:show_data)
-  post '/add/data', to: DailyDataController.action(:add_data)
+  match '/show/data', to: 'daily_data#show_data', via: :get
+  match '/add/data', to: 'daily_data#add_data', via: :post
   match '*path', via: :all, to: DailyDataController.action(:all_paths)
```

Comment:
You can replace `DailyDataController.action(:all_paths)` with shorthand syntax too.